src/main/minio.js

```diff
@@ -2259,6 +2259,246 @@ export class Client {
     this.makeRequest({method, bucketName, query}, payload, 200, '', false, cb)
   }
 
+  /** To set Tags on a bucket or object based on the params
+     *  __Arguments__
+     * taggingParams _object_ Which contains the following properties
+     *  bucketName _string_,
+     *  objectName _string_ (Optional),
+     *  tags _object_ of the form {'<tag-key-1>':'<tag-value-1>','<tag-key-2>':'<tag-value-2>'}
+     *  putOpts _object_ (Optional) e.g {versionId:"my-object-version-id"},
+     *  cb(error)` _function_ - callback function with `err` as the error argument. `err` is null if the operation is successful.
+     */
+  setTagging(taggingParams){
```

Comment:
```suggestion
  setObjectTagging(taggingParams){
```

Reply:
@kannappanr  this in an internal method.  

`putObjectTagging` is the method exposed as API. 

I have followed the naming of methods similar to go sdk.

Reply:
I have updated to `setObjectTagging` and the examples as well.